recipes/cgns/all/conandata.yml

```diff
@@ -1,10 +1,8 @@
 sources:
-  "3.4.1":
```

Comment:
Could you please explain why did you need to drop 3.4.1?

Reply:
I couldn't make it work with conan v1-v2 transition, I'm close to giving up ...
I am happy to not drop 3.4.1 but I can't keep spending time bashing my head against walls.

It is probably something very simple for someone else to fix...